core/src/main/java/nl/nn/adapterframework/core/SharedWsdlDefinitions.java

```diff
@@ -21,7 +21,10 @@
 import nl.nn.adapterframework.configuration.ConfigurationException;
 import nl.nn.adapterframework.functional.ThrowingFunction;
 
-public class SharedResources<R> {
+/**
+ * This exists because WSDL Definitions can grow dramatically in size when they include a few xsd's.
+ */
+public class SharedWsdlDefinitions<R> {
 
 	private Map<String, R> resources = new HashMap<>();
```

Comment:
In plaats van hier een type-parameter `R` kan je ook de class `Definition` gebruiken, die uiteindelijk wordt gebruikt. En dan kan je de type-parameter weghalen uit de class-definitie.